css/styles.css

```diff
@@ -35,4 +34,52 @@ h3 {
 button {
     font-size: 0.8125rem; /* 0.8125 * 16 - 13px */
     font-weight: 700;
-}
\ No newline at end of file
+}
+
+
+
+/*** 2.0 - Header Styles ***/
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 20px;
+    height: 80px;
+    margin: 0 80px;
```

Comment:
Same as other comment. Use rems or ems. Preferably ems for padding/margins and rems for font size.